src/Dialogs/Settings/Panels/SiteConfigPanel.cpp

```diff
@@ -31,69 +30,72 @@ class SiteConfigPanel final : public RowFormWidget {
   };
 
 public:
-  SiteConfigPanel()
-    :RowFormWidget(UIGlobals::GetDialogLook()) {}
+  SiteConfigPanel() : RowFormWidget(UIGlobals::GetDialogLook()) {}
```

Comment:
Please keep on same line. (I know 80 character limit, but it isn´t a hard limit if it enhances readability.)